### ISSUE TITLE
[Bug]: Cannot set properties of undefined (setting 'value') is not resolved.

### ISSUE THREAD

**Issue by HikaruShiho** [ueberdosis/tiptap#5630]
### Affected Packages

@tiptap/vue-3, @tiptap/starter-kit

### Version(s)

2.7.1

## development environment
- vue: `3.4.37`
- typescript: `5.5.4`
- node: `20.11.1`
- tiptap: `2.7.1`

## What problems do you have?
[Described in the official documentation.](https://tiptap.dev/docs/editor/getting-started/install/vue3)I am trying to display the editor in a way that is not feasible and I am having trouble achieving that.

I am trying to display the editor with the following code.
```javascript
// Editor.vue
<script setup lang="ts">
import { useEditor, EditorContent } from '@tiptap/vue-3'
import StarterKit from '@tiptap/starter-kit';

const emit = defineEmits<{ (e: "update:modelValue", content: string): void }>();

const editor = useEditor({
  element: document.querySelector('#editor-content') as Element,
  extensions: [StarterKit],
  onUpdate({ editor }) {
    emit('update:modelValue', editor.getHTML())
  },
  content: '<p>I’m running Tiptap with Vue.js. 🎉</p>',
})
</script>

<template>
  <EditorContent id="editor-content" v-if="editor" :editor="editor" />
</template>
```

```javascript
// Parenrt.vue
<script setup lang="ts">
import Editor from "@/components/atoms/Editor.vue";

const content = ref("");
</script>

<template>
  <div class="editor-wrap">
    <Editor v-model="content" />
  </div>
</template>
```

However, when I check with my browser, I get the following error message and the editor is not displayed.

![スクリーンショット 2024-09-18 16 58 16](https://github.com/user-attachments/assets/bd6b56a8-3936-4a31-b863-10d5a56d616b)

```javascript
  /**
   * Register a ProseMirror plugin.
   */
  registerPlugin(plugin, handlePlugins) {
    super.registerPlugin(plugin, handlePlugins);
    this.reactiveState.value = this.view.state; // here
  }
```
According to the error, it seems that the error is due to a missing value for `this.view.state` inside the `registerPlugin` method, or is the `useEditor` property missing?

I would appreciate your advice.

### Browser Used

Chrome

### Expected Behavior

I hope the editor will display correctly

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by MCYouks** [comment]
Same issue here. Thanks for reporting it

**Comment by nperez0111** [comment]
I was able to get this to run properly with 

```vue
<script setup lang="ts">
import StarterKit from '@tiptap/starter-kit'
import { EditorContent, useEditor } from '@tiptap/vue-3'

const emit = defineEmits<{(e: 'update:modelValue', content: string): void }>()

const editor = useEditor({
  // element: document.querySelector('#editor-content') as Element,
  extensions: [StarterKit],
  onUpdate({ editor }) {
    emit('update:modelValue', editor.getHTML())
  },
  content: '<p>I’m running Tiptap with Vue.js. 🎉</p>',
})

</script>

<template>
  <EditorContent id="editor-content" v-if="editor" :editor="editor" />
</template>
```
by commenting out the `element` which is not needed since it is bound to the editor content.

If you still run into issues can you make a codesandbox?

**Comment by DenisOstr** [comment]
```ts
const editor = useEditor({
    extensions: [StarterKit.configure({
      codeBlock: false
    }), Underline, CodeBlockLowlight.configure({
      lowlight: createLowlight(all),
    })],
    content: 'Content here',
  })
```

Same error when extensions appears:
```ts
CodeBlockLowlight.configure({
    lowlight: createLowlight(all),
})
```

**Comment by HikaruShiho** [comment]
@nperez0111 
Thanks for the advice!
I tried commenting out the element but there was no change.

[codesandbox](https://codesandbox.io/p/devbox/tiptap-demo-g8xm2s) created.
pleased to meet you.

**Comment by HikaruShiho** [comment]
@nperez0111 
When I downgraded the version to `“@tiptap/vue-3”:“^2.6.6”,` the editor appeared without problems.

**Comment by fushugaku** [comment]
Same thing is happening for us

**Comment by xzgun89425** [comment]
I have same problem, Did anyone sloved the bug?

**Comment by DenisOstr** [comment]
> Same thing is happening for us this.reactiveState.value is undefined while setting paste plugin <img alt="image" width="459" src="https://private-user-images.githubusercontent.com/1581077/369607311-5950c146-771c-42ed-9231-99ef4e37d1ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5MTcyMDAsIm5iZiI6MTcyNjkxNjkwMCwicGF0aCI6Ii8xNTgxMDc3LzM2OTYwNzMxMS01OTUwYzE0Ni03NzFjLTQyZWQtOTIzMS05OWVmNGUzN2QxY2EucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjFUMTEwODIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmEzYzBhY2IzZGM5MGVkMTgwNTZkOTU0Y2VmNjhkMWNiZTZmZjIxMjUxZjgxZDAyYzY1YTc3YTk0OGZkMTg1NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.jTJLF1_XwrWE0m2zK4G5mYTWSjzOQmMynKnbgRNFRTM">
> 
> <img alt="image" width="386" src="https://private-user-images.githubusercontent.com/1581077/369607277-80a04bdf-b4fe-4713-88ab-8ce106469b1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5MTcyMDAsIm5iZiI6MTcyNjkxNjkwMCwicGF0aCI6Ii8xNTgxMDc3LzM2OTYwNzI3Ny04MGEwNGJkZi1iNGZlLTQ3MTMtODhhYi04Y2UxMDY0NjliMWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjFUMTEwODIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmRhYTc0YjkwMThlOTJhZTBkYmIwYjQ4ZmUxM2YxMjY5MjExMzA5YTY1ZTA5ZWY3Y2M1Y2M4NzYxMzY2MGM4MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.8Xa7cJodt5S5AI2CLpJth5UY3OINzkZAHgxPivuJhU0">

https://github.com/ueberdosis/tiptap/issues/5630#issuecomment-2359904938

**Comment by DenisOstr** [comment]
> I have same problem, Did anyone sloved the bug?

https://github.com/ueberdosis/tiptap/issues/5630#issuecomment-2359904938

**Comment by herlandroando** [comment]
I got this bug too, but I tried to update all dependencies related to tiptap such as its core and extensions to v2.7.2. Finally I don't get this error anymore and still no problem for now.

This is my package.json for tiptap related libraries:
![image](https://github.com/user-attachments/assets/68b4d924-43af-4692-9ca4-5fffa5c89ca3)

**Comment by xzgun89425** [comment]
@herlandroando you slove my problem, thanks !!

**Comment by fushugaku** [comment]
updating to 2.7.2 also solved the problem, @herlandroando, thank.you

**Comment by nperez0111** [comment]
Alright I will close this as resolved by my 2.7.2 update
